gridscale/resource_gridscale_server.go

```diff
@@ -536,7 +536,7 @@ func resourceGridscaleServerUpdate(d *schema.ResourceData, meta interface{}) err
 
 	//The ShutdownServer command will check if the server is running and shut it down if it is running, so no extra checks are needed here
 	if shutdownRequired {
-		err = client.ShutdownServer(emptyCtx, d.Id())
+		err = client.StopServer(emptyCtx, d.Id())
```

Comment:
Why did you replace `ShutdownServer` with `StopServer`? `StopServer` sends an ACPI shutdown signal to the virtual machine, which does like the power button of a computer. `ShutdownServer` will do graceful power-off.

Reply:
Currently ACPI shutdown does not work (I've tested)

Reply:
plz, keep `ShutdownServer`.  I tested it from the panel and worked well. if you have any error, report that in the developers' slack channel.

Reply:
Decreasing number of CPUs just needs the server to be stopped btw. I don't see why we need to shut it down when we just need to stop it.

Reply:
both will stop the server. `StopServer ` does not consider the running processes. If a vm is writing to disk when it receives a Power Off command, data corruption may occur.